text/0000-project-descriptor.md

```diff
@@ -0,0 +1,281 @@
+# Meta
+[meta]: #meta
+- Name: Project Descriptor
+- Start Date: 2019-06-11
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: https://github.com/buildpack/rfcs/pull/16
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new file, `project.toml`, containing configuration for apps, services, functions, and buildpacks.
+
+# Motivation
+[motivation]: #motivation
+
+This proposal is meant to address all of the following issues:
+
+* Build metadata (i.e. information about the code being built)
+* [Monorepos](https://github.com/buildpack/spec/issues/39)
+* [Application descriptor](https://github.com/buildpack/spec/issues/44)
+* [Ignoring files](https://github.com/buildpack/pack/issues/210)
+* Multiple images (i.e. one build, multiple images as output)
+* Custom launch configuration
+* Codified buildpacks
+
+# What it is
+[what-it-is]: #what-it-is
+
+Terminology:
+
+* **project**: a repository containing source code for an app, service, function, buildpack or a monorepo containing any combination of those.
+* **image**: the output of a running a buildpack(s) against a project
+
+The target personas for this proposal is buildpack users who need to enrich or configure buildpack execution. The elements in  `project.toml` support many different use cases including three top level tables:
+
+- `[project]`: (optional) defines configuration for a project
+- `[[images]]`: (optional) defines configuration for an image
+- `[metadata]`: (optional) metadata about the repository
+
+Here is an overview of the complete schema:
+
+```toml
+[project]
+id = "<string>"
+name = "<string>"
+version = "<string>"
+authors = ["<string>"]
+documentation = "<url>"
+license = "<string>"
+source = "<url>"
+include = ["<string>"]
+exclude = ["<string>"]
+publish = "<boolean>"
+
+[[images]]
+id = "<string>"
+path = "."
+stack = "<string>"
+
+  [[images.launch.env]]
+  name = "<string>"
+
+  [[images.build.env]]
+  name = "<string>"
```

Comment:
Should we add `value = "<string>"` to these two?

Reply:
yes 🤦‍♂️